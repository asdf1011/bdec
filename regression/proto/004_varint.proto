message Varints {
    required int32 a = 1;
    required int64 b = 2;
    required uint32 c = 3;
    required uint64 d = 4;
    required sint32 e = 5;
    required sint64 f = 6;
}
