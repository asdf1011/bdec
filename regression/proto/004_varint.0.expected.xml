<Varints>
    <a>2147483647</a>
    <b>9223372036854775807</b>
    <c>4294967295</c>
    <d>18446744073709551615</d>
    <e>2147483647</e>
    <f>9223372036854775807</f>
</Varints>
